Sample/DIDChat/DIDChat/Modules/MediatorView/MediatorViewModel.swift

```diff
@@ -36,7 +36,10 @@ class MediatorViewModelImpl: MediatorViewModel {
                 await MainActor.run { [weak self] in
                     self?.routeToContactsList = true
                 }
-            } catch {}
+            } catch {
+                print(error)
+                print()
```

Comment:
Remove Print from code.
```suggestion
```

Reply:
One of them I will leave since it is in the demo ;)